Lab Rat - GWJ 2020/Scenes/Level/grundgerüst.tscn

```diff
@@ -8,9 +8,10 @@
 [node name="Spatial" type="Spatial"]
 
 [node name="Camera_player" parent="." instance=ExtResource( 4 )]
-
 transform = Transform( 1, 0, 0, 0, 1, 0, 0, 0, 1, -0.0199528, -3.8147e-06, -0.0360804 )
 
+[node name="DirectionalLight" type="DirectionalLight" parent="Camera_player"]
```

Comment:
Aus welchem Grund hast du das DirectionalLight nun dem "Camera_Player" angehängt?

Reply:
Kann ich wieder zurückändern. Das DirectionalLight hat sowieso keine Position, sondern nur einen Winkel. War, um einen Winkel zu finden, in dem man keinen Lichtkegel auf dem Labyrinthboden sieht.

Reply:
Ok, also damit man keinen Lichtkegel sieht, muss entweder das Material bei metallic odwr roughness geändert werden oder das directional light rotiert mit der Kamera. Das hat man vorher nicht gesehen, weil das Licht disabled war.

Reply:
was hast du denn gegen eine Lichtkegel? und deine Art das zu lösen, indem man es mit dem player mitdreht finde ich ein bisschen komisch. ist nicht gaaaannz so realistisch

Reply:
DirectionalLight = Sonne, das steht immer extra 😅

Reply:
> was hast du denn gegen eine Lichtkegel? und deine Art das zu lösen, indem man es mit dem player mitdreht finde ich ein bisschen komisch. ist nicht gaaaannz so realistisch

Aber soll das wirklich so aussehen?
![Lichtkegel](https://user-images.githubusercontent.com/35505481/79379010-8009ca00-7f5e-11ea-8c95-f071bc0703b6.png)

Reply:
ich finde es nicht so schlimm. das sieht einfach realistischer aus, finde ich. 
ps: lila licht sieht jetzt nicht soooo genial aus

Reply:
Ok, habe jetzt roughness und metallic beim Material geändert, jetzt geht das mit dem directional light wieder (ist jetzt wieder unter dem Spatial und nicht unter der Kamera).

Reply:
Das level bekommt eh noch eine andere textur und vorallem .. mir fällt gerade mal auf, es ist im labor, also da sollte eh keine sonne sein :D